packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -94,7 +95,19 @@ export const MafsGraph = React.forwardRef<
                     width={width}
                     height={height}
                 >
+                    {/* Background layer */}
                     {!legacyGrid && <Grid {...props} />}
+
+                    {/* Locked layer */}
+                    {props.lockedFigures && (
+                        <MafsLockedLayer
+                            // Make a copy of lockedFigures since the prop
+                            // type is readonly.
```

Comment:
Is `lockedFigures` still readonly? The type is `Array<LockedFigure>`, so I think you should be able to use the array without copying it.

Reply:
The problem is that `lockedFigures` is readonly in perseus-types.ts. I can. change it to not be readonly though